tool/tsh/tsh.go

```diff
@@ -939,6 +954,27 @@ func onBenchmark(cf *CLIConf) {
 		fmt.Fprintln(os.Stderr, utils.UserMessageFromError(err))
 		os.Exit(255)
 	}
+
+	if cf.BenchExport {
+		fo, err := getLatencyProfileFile(cf.BenchExportPath)
+		if err != nil {
+			fmt.Printf("%v", err)
+			return
+		}
+		defer func() {
+			if err := fo.Close(); err != nil {
+				fmt.Fprintf(os.Stderr, "failed saving latency profile to %q: %s\n", fo.Name(), utils.UserMessageFromError(err))
+				return
+			}
+			fmt.Printf("Latency profile saved: %v\n", fo.Name())
+		}()
+
+		if err := writeHistogram(fo, result.Histogram, cf.BenchTicks, cf.BenchValueScale); err != nil {
+			fmt.Fprintln(os.Stderr, trace.UserMessage(err))
+			return
+		}
```

Comment:
wrap all of this into a single function, like `func exportLatencyProfile(cf *CLIConf) error` (it can replace `getLatencyProfileFile` instead of calling it)

Reply:
CLIConf does not contain the benchmark result, so I need to call a function from `onBenchmark`